### PR TITLE
Replace Travis by Github Actions

### PR DESCRIPTION
**Pull request by eXorus**
See https://github.com/Codeception/Codeception/issues/5890

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  tests:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        php: [7.1, 7.2, 7.3, 7.4]
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v1
```
Comment (Slamdunk) [inline .github/workflows/main.yml]:
```suggestion
      uses: actions/checkout@v2
```

File: .github/workflows/main.yml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+      run: |
+        git clone -q --depth=1 -b master --recurse-submodules https://github.com/Slamdunk/codeception-mezzio-tests framework-tests
+        git --git-dir framework-tests/.git log -n 1
+        composer update -d framework-tests --no-dev --prefer-dist --no-interaction
+
+    - name: Validate composer.json and composer.lock
+      run: composer validate
+
+    - name: Install dependencies
+      run: composer install --prefer-dist --no-progress --no-suggest
```
Comment (Slamdunk) [inline .github/workflows/main.yml]:
```suggestion
      run: composer install --prefer-dist --no-progress --no-suggest  --no-interaction
```

**Review by Slamdunk — CHANGES_REQUESTED** [review]
Awesome! Just few changes and we're ready to go :+1:

**Comment by eXorus** [comment]
@Slamdunk Thanks for the review, I will update all the Github Actions for all the Codeception Packages.
https://github.com/Codeception/Codeception/issues/5890

I didn't finish yet.

**Comment by Slamdunk** [comment]
> I didn't finish yet.

Ok. Next time so try the Draft PR button: https://github.blog/2019-02-14-introducing-draft-pull-requests/

**Comment by eXorus** [comment]
Nice I didn't know that

**Pull request merged**